mrack.spec

```diff
@@ -0,0 +1,61 @@
+# Created by pyp2rpm-3.3.5
+%global srcname mrack
+
+Name:           %{srcname}
+Version:        0.6.0
+Release:        1%{?dist}
+Summary:        Multicloud use-case based multihost async provisioner for CIs and testing during development
+
+License:        Apache License 2.0
+URL:            https://github.com/neoave/mrack
+Source0:        https://github.com/neoave/mrack/releases/download/v%{version}/mrack-%{version}.tar.gz
+BuildArch:      noarch
+
+### Patches ###
+Patch0001:  0001-chore-set-boto-minimum-versions.patch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(asyncopenstackclient) >= 0.8.1
```

Comment:
If I understand https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/ correctly, packager has an option to specify rpm deps directly e.g. python3-asyncopenstackclient or use this - "automatically generated dependencies". When automatically generated deps are used then the requiremnts.txt is checked. This spec file is using version 0.6.0 so it gets it from the released version and not git repo - thus the reason for the Patch0001. 

So I think we have the following options:
- do not use automatically generated deps 
- use the automatically generarated deps, but have corrected requirements.txt (which will happen only after this patch merge and a release).

Reply:
"do not use automatically generated deps" -- I preferred this approach.